### ISSUE TITLE
[1.19.2] Broken block lighting on >= Forge 43.1.24

### ISSUE THREAD

**Issue by strdsk** [Asek3/Rubidium#311]
Forge changelog:
```
- 43.1.24 Allow faces of an "elements" model to have disabled ambient occlusion (#9019)
```

All block surface becomes black.
Will upload an attachment later, if necessary.

**Comment by Asek3** [comment]
Was fixed in 0.6.2

**Comment by strdsk** [comment]
Sorry, didn't realize that ferium actually not downloading the latest version, hence, here i am :/
